snapshot/snapshot.go

```diff
@@ -293,10 +293,14 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 	if label.IsNydusMetaLayer(info.Labels) {
 		err = o.fs.WaitUntilReady(id)
 		if err != nil {
-			return nil, errors.Wrapf(err, "mounts: snapshot %s is not ready, err: %v", id, err)
+			// Skip waiting if clients is unpacking nydus artifacts to `mounts`
```

Comment:
Can we explain when & where it happens in this comment? So as not to confuse readers in the future.

Reply:
Sure. Will add a more detailed description